polaris-patterns/src/components/utilities/dates/index.ts

```diff
@@ -0,0 +1,64 @@
+const VALID_YYYY_MM_DD_DATE_REGEX = /^\d{4}-\d{1,2}-\d{1,2}$/;
```

Comment:
These utilities are copied from web. Do we like the utilities here in `src/components/utilities` or should it be outside in `src/utilities`?